packages/manager/apps/web-ongoing-operations/src/data/api/ongoing-operations-actions.ts

```diff
@@ -0,0 +1,15 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { AxiosResponse } from 'axios';
+import { OperationName } from '@/enum/operationName.enum';
+
+//  Actions
+export const updateOperationStatus = async (
+  universe: string,
+  id: number,
+  operationName:
+    | OperationName.CanAccelerate
+    | OperationName.CanCancel
+    | OperationName.CanRelaunch,
+): Promise<AxiosResponse> => {
+  return apiClient.v6.post(`/me/task/${universe}/${id}/${operationName}`);
+};
```

Comment:
```suggestion
import { v6 } from '@ovh-ux/manager-core-api';
import { AxiosResponse } from 'axios';
import { OperationName } from '@/enum/operationName.enum';

//  Actions
export const updateOperationStatus = async (
  universe: string,
  id: number,
  operationName:
    | OperationName.CanAccelerate
    | OperationName.CanCancel
    | OperationName.CanRelaunch,
): Promise<AxiosResponse> => {
  const {data} = await v6.post(`/me/task/${universe}/${id}/${operationName}`);
  return data;
};
```

Reply:
For all apiClient you can directly use v6.
If you use async function, you need to have await if not it isn't needed to have async function.